### DISCUSSION TITLE
Dodatkowe pola w GET /sale/offers

### DISCUSSION THREAD

**Discussion by TomaszSzydlowski-sudo** [allegro/allegro-api#4352] [Suggestion - verification]
Proponuję dodanie pól na endpoincie GET /sale/offers:
-  "handlingTime" w sekcji "delivery" - Przyda się do sprawdzenia jakie oferty nie mają np. PT24H
- sekcji "promotion" - Żeby sprawdzić czy dana oferta jest promowana trzeba odpytać /sale/offers/{id} dla każdej oferty 

Pozdrawiam

**Comment by FilipSerewa** [comment]
handlingTime byłby bardzo pomocny - aktualnie jak chcę pobrać ten parametr dla 150 tyś aukcji - muszę wywołać tyle razy metodę sale/product-offers/ co zajmuje wieczność...
